rechnung/cli.py

```diff
@@ -59,28 +72,32 @@ def render():
     Render all unrendered invoices.
```

Comment:
Needs to be updated to reflect that it, in fact, renders both, invoices and contracts. :)